scripts/monitoring/ops-metric-client.py

```diff
@@ -64,6 +64,14 @@ def run(self):
 
         self.metric_sender.send_metrics()
 
+    @staticmethod
+    def adjust_type(val):
+        """ convert float values to float type. keep string as is """
+        try:
+            return float(val)
```

Comment:
@zgalor @twiest any way to know the type in advance instead of checking if something can be transformed into a float or not?